Formula/sox.rb

```diff
@@ -3,9 +3,13 @@ class Sox < Formula
   homepage "https://sox.sourceforge.io/"
   url "https://downloads.sourceforge.net/project/sox/sox/14.4.2/sox-14.4.2.tar.gz"
   sha256 "b45f598643ffbd8e363ff24d61166ccec4836fea6d3888881b8df53e3bb55f6c"
-  # This has been found on 2023-07-10 on https://sourceforge.net/projects/sox/
-  # under the headline of "License"
-  license "GPL-2.0-only"
+  # This is based on  https://sourceforge.net/projects/sox/, as found
+  # under the headline of "License" (citation):
+  #   GNU Library or Lesser General Public License version 2.0 (LGPLv2),
+  #   GNU General Public License version 2.0 (GPLv2)
+  # (as of 2023-07-11)
+  # which translates into:
```

Comment:
No need for this comment since the discussion is in this pull request already.

Reply:
Okay, uncluttering the formula is done.